### PR TITLE
Fix movielist

### PR DESCRIPTION
**Pull request by gosiagajlusz**
I made like 20 previews,which were good,  so I hope this time it remains good looking on every device :)

### CODE REVIEW

File: src/common/GlobalStyle.js [hunk 1]
```diff
@@ -12,6 +12,12 @@ export const GlobalStyle = createGlobalStyle`
     body {
         font-family: 'Poppins', sans-serif;
         color: ${({ theme }) => theme.colors.textPrimary};
-        background: ${({ theme }) => theme.colors.background};
-    }
-`;
\ No newline at end of file
+        background: ${({ theme }) => theme.colors.background};     
+        margin:0 auto;          
+    display:flex;
```
Comment (mombil) [inline src/common/GlobalStyle.js:17]: Why you put display flex to GlobalStyle? These doesn't look right

File: src/common/Header/styled.js [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
-export const StyledHeader = styled.header`
+export const StyledHeader = styled.h1`
   font-size: 36px;
   font-weight: 600;
   line-height: 43.2 px;
   max-height: 45px;
-  margin-left: 276px;
-  margin-top: 56px;
+  text-align: left;
+  /* justify-self: start; */
```
Comment (mombil) [inline src/common/Header/styled.js:9]: is this comment really necessary

File: src/features/MovieList/MovieList.js [hunk 3]
```diff
@@ -17,17 +20,22 @@ const MovieList = () => {
...
+            ? Array.from({ length: 8 }).map((_, index) => (
+                <Tile key={index}>miejsce na film</Tile>
+              ))
+            : movies.map((movie, index) => (
+                <Tile key={index}>
+                  <MovieCard {...movie} />
+                </Tile>
+              ))}
+        </TilesWrapper>
+        <Pagination />
```
Comment (mombil) [inline src/features/MovieList/MovieList.js:37]: Pagination is back😄

**Review by mombil — CHANGES_REQUESTED** [review]
I think you should definitely change display flex and other styles with flex because not every element will need display flex in the future

File: src/common/GlobalStyle.js (hunk 1 above)
Comment (gosiagajlusz) [reply src/common/GlobalStyle.js:17]: ok, in the end it wasn't necessary.

File: src/features/MovieList/MovieList.js [hunk 4]
```diff
@@ -17,17 +20,22 @@ const MovieList = () => {
...
-              <Tile key={index}>miejsce na film</Tile>
-            ))
-          : movies.map((movie, index) => (
-              <Tile key={index}>
-                <MovieCard {...movie} />
-              </Tile>
-            ))}
-      </TilesWrapper>
+      <Container>
+        <Header title="Popular movies"></Header>
```
[the remark this answers is not in the document]
Comment (gosiagajlusz) [reply src/features/MovieList/MovieList.js:24]: what you mean?

[the remark this answers is not in the document]
Comment (mombil) [reply src/features/MovieList/MovieList.js:24]: sory i mean you can just give title beetwen header no as parameter

[the remark this answers is not in the document]
Comment (gosiagajlusz) [reply src/features/MovieList/MovieList.js:24]: Please explain what makes your solution better then current? Or what's wrong with current solution?

**Comment by mombil** [comment]
Now it's look very good

**Pull request merged**